### PR TITLE
Introduce sizing tool to documentation page.

### PR DESCRIPTION
**Pull request by jeschkies**
**What this PR does / why we need it**:
The sizing tool should help to find a good cluster sizing when starting out with Loki. It's a dynamic form withing the documentation.

image

This pull request will be followed by help on hover.

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Reviewed the `CONTRIBUTING.md` guide
- [x] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

**Review by MasslessParticle — COMMENTED** [review]
Nice work. This should be awesome.

I see we've got some commented out code in `calculateClusterSize` and in `node.go`, can that be deleted?

Some tests would be awesome, particularly around the mathy parts, to help future engineers who work on this.

File: pkg/sizing/algorithm_test.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		if ingest < 0 {
+			ingest = -ingest
+		}
+		replicas := true
+		for _, cloud := range NodeTypesByProvider {
+			for _, node := range cloud {
+				size := calculateClusterSize(node, ingest, Basic)
+				replicas = size.totalNodes > 0.0 && size.totalReadReplicas > 0.0 && size.totalWriteReplicas > 0.0
+				require.Truef(t, replicas, "Cluster size was empty: ingest=%d cluster=%v node=%v", ingest, size, node)
+				require.InDelta(t, size.totalReadReplicas, size.totalWriteReplicas, 5.0, "Replicas have different sizes: ingest=%d node=%s", ingest, node.name)
```
Comment (jeschkies) [inline pkg/sizing/algorithm_test.go:21]: This assumes that the replicas should be about equal. For very large input they are not. Interestingly the algorithm requires  >3000 replicas if the ingest is 1 Petabyte.

**Review by MasslessParticle — APPROVED** [review]

**Comment by sathieu** [comment]
@jeschkies The webpage is referencing localhost:

```
const API_URL = `http://localhost:3001/api/sizing`
```

**Comment by jeschkies** [comment]
@sathieu it's been fixed. https://grafana.com/docs/loki/latest/installation/sizing/ should work

**Comment by sathieu** [comment]
Thanks @jeschkies.

**Pull request merged**